service/gcs/runtime/runc/runc.go

```diff
@@ -587,7 +587,11 @@ func (c *container) startProcess(tempProcessDir string, hasTerminal bool, stdioS
 	}
 
 	if err := cmd.Run(); err != nil {
-		return nil, errors.Wrapf(err, "failed to run runc create/exec call for container %s", c.id)
+		b, readErr := ioutil.ReadFile(logPath)
```

Comment:
@jterry75 Can you provide feedback on this? The only way I could find to get the actual error logged to stderr from running the cmd was by reading the runc log that's in the pod. This log file includes info like log level and timestamp of the message. What are you thoughts on simply logging in opengcs the runc file from the pod instead of returning it in the error to startProcess?

Reply:
Updated code with suggestions from John.